.gitignore

```diff
@@ -42,7 +42,7 @@ reports_webapp.pid
 
 # Config files
 universe_wsgi.ini
-reports_wsgi.ini
+reports.ini
```

Comment:
Do we have to replace `<universe_wsgi.ini>` everywhere since it is replaced by `<galaxy.ini>`,as mentioned by @martenson?